### ISSUE TITLE
🐛 BUG: don't try to interpret commented out `@import`s in css

### ISSUE THREAD

**Issue by stefanprobst** [snowpackjs/astro#1490]
### What package manager are you using?

yarn

### What operating system are you using?

linux

### Describe the Bug

a commented out `@import` at-rule in a global css file, like this:

```css
/* @import '@csstools/normalize.css'; */
```

will produce the following error:

```
(node:1721654) UnhandledPromiseRejectionWarning: Error: Module "@csstools/normalize.css" not found.
    If you‘re trying to CSS file from your project, try "./@csstools/normalize.css".
    If you‘re trying to import an NPM package, try running `npm install @csstools/normalize.css` and re-running Snowpack.
    at Object.resolveEntrypoint (/data/Development/playground/astro-issue-css-comments/node_modules/esinstall/lib/entrypoints.js:171:19)
    at PackageSourceLocal.buildPackageImport (/data/Development/playground/astro-issue-css-comments/node_modules/snowpack/lib/cjs/sources/local.js:424:40)
    at async PackageSourceLocal.prepare (/data/Development/playground/astro-issue-css-comments/node_modules/snowpack/lib/cjs/sources/local.js:227:13)
    at async startServer (/data/Development/playground/astro-issue-css-comments/node_modules/snowpack/lib/cjs/commands/dev.js:240:9)
    at async createSnowpack (file:///data/Development/playground/astro-issue-css-comments/node_modules/astro/dist/runtime.js:283:14)
    at async createRuntime (file:///data/Development/playground/astro-issue-css-comments/node_modules/astro/dist/runtime.js:303:7)
    at async dev (file:///data/Development/playground/astro-issue-css-comments/node_modules/astro/dist/dev.js:17:19)
```

### Steps to Reproduce

1. `git clone https://github.com/stefanprobst/astro-issue-css-comments.git && cd astro-issue-css-comments`
2. `yarn && yarn build`
3. see above error message

### Link to Minimal Reproducible Example (Optional)

https://github.com/stefanprobst/astro-issue-css-comments.git

**Comment by stefanprobst** [comment]
interestingly when removing the semicolon it's fine, i.e. the following is ok:

```css
/* @import '@csstools/normalize.css' */
```

**Comment by natemoo-re** [comment]
Err... will close when `astro@next` is released as stable. Thanks!

**Comment by natemoo-re** [comment]
Following up on my previous comment, the team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
